### PR TITLE
Update to latest swagger models

### PR DESCRIPTION
**Pull request by zcross**
This includes support for the new `dataset` entity

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 ## Unreleased
 
+## v1.6.0
```
Comment (prashantv) [inline CHANGELOG.md:5]: intentional?

File: src/generated/cli/configv1/datasets.gen.go [hunk 2]
```diff
@@ -0,0 +1,611 @@
...
+	}
+
+	listOptions.registerFlags(cmd.Flags())
+	clientFlags.AddFlags(cmd)
+	outputFlags.AddFlags(cmd)
+
+	return cmd
+}
+
+const DatasetScaffoldYAML = `api_version: v1/config
```
Comment (bg451) [inline src/generated/cli/configv1/datasets.gen.go:495]: @zcross FYI this is where the scaffold bits are

**Review by bg451 — APPROVED** [review]
lgtm pending prashant's comment on the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (zcross) [reply CHANGELOG.md:5]: ah, I see. thanks!

**Pull request merged**